sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -112,6 +112,17 @@ internal FieldValue(SelectionMarkState value)
             ValueSelectionMark = value;
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FieldValue"/> structure.
```

Comment:
nit: could u add to all these methods that they are used for mocking. Last time it took me a while to understand if this were for mocking or used by our generator